src/main/java/roomescape/Domain/Reservation.java

```diff
@@ -37,24 +34,15 @@ public String getDate() {
         return date;
     }
 
-    public String getTime() {
+    public Time getTime() {
         return time;
     }
 
-    public void setId(Long id)
-    {
-        this.id = id;
-    }
-
-    public static Reservation toEntity(Reservation reservation, long id) {
-        return new Reservation(id, reservation.name, reservation.date, reservation.time);
-    }
-
-    public boolean isValid() {
-        return isNotNullOrEmpty(name) && isNotNullOrEmpty(date) && isNotNullOrEmpty(time);
+    public boolean notEmpty() {
+        return notNullOrEmpty(name) && notNullOrEmpty(date) && time != null;
     }
```

Comment:
null check를 위에서 해주면 아래에서는 굳이 notNull... 처럼 메소드를 처리하지 않아도 되지 않을까요?
```suggestion
    public boolean isEmpty() {
        if(name == null || date == null || time == null) return false;
        return name.isEmpty();
    }
```

Reply:
null Check의 수정이나 예외처리에 관한 구문에 대한 처리가 더 필요할 것 같습니다!
추가 작성 해보겠습니다!😄